### ISSUE TITLE
0.3.0 bug: "wp_" Hard coded as table prefix, breaks sites on upgrade

### ISSUE THREAD

**Issue by paulisloud** [wp-graphql/wp-graphql#785]
Quick fix, in src/Data/Config.php, I'd suggest:

```
public function graphql_wp_query_cursor_pagination_stability( $orderby ) {
	global $wpdb;
	$prefix = $wpdb->prefix;
	return $orderby . ', ' . $prefix . 'posts.ID DESC ';
}
```

**Comment by epeli** [comment]
Sorry by bad. Sent a fix here https://github.com/wp-graphql/wp-graphql/pull/787

**Comment by jasonbahl** [comment]
@paulisloud @epeli thanks! reviewing 👀

**Comment by jasonbahl** [comment]
I merged this, then realized it still had a bug. Merged #790 as a follow up, then released v0.3.01 as this bug was causing some bad issues to folks dashboards. 

Thanks for the quick report @paulisloud  and the quick response @epeli!
